pilot/pkg/networking/util/util.go

```diff
@@ -37,6 +37,7 @@ import (
 	pstruct "github.com/golang/protobuf/ptypes/struct"
 	"github.com/golang/protobuf/ptypes/wrappers"
 	"github.com/hashicorp/go-multierror"
+	"google.golang.org/protobuf/types/known/structpb"
```

Comment:
We are using this lib "github.com/golang/protobuf/ptypes/struct"

Reply:
Thanks! done.